testbed/addons/kit/construct.ts

```diff
@@ -92,5 +71,27 @@ export class Kit extends cdk.Construct {
             }
         })
         chart.node.addDependency(ns)
+
+        //Karp Provisioner for kit
```

Comment:
I think this should stay in Karpenter/construct.ts as this is a default provisioner, if we were to create KIT specific provisioner then it would make sense to have it here.

Reply:
This provisioner is specifically only used for KIT CP, as KIT is leveraging Karp, right ? given it's subnetSelector catered to KIT operator. It makes sense to be here for me, wdyt ?